rest/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.16.20</version>
+            <version>1.18.4</version>
```

Comment:
i agree with updating things, but it is subtle and not described in this commit message.